packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -174,6 +175,7 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
             incrementDirection: 1,
             usePageStep: false,
             direction: Direction.ltr,
```

Comment:
I think you can avoid adding an additional state property by defaulting the `direction` state property to `null`. Then the check in render becomes "do I know my direction? If not, don't render anything", which feels more in line with the purpose of this change.

Reply:
🔥 🔥 🔥 - I like this.

Reply:
yup, good call